contrib/bentoml/OWNERS

```diff
@@ -1,2 +1,3 @@
 approvers:
   - yubozhao
+  - juliusvonkohout
```

Comment:
Based on [contrib guideline](https://github.com/kubeflow/manifests/blob/master/proposals/20220926-contrib-component-guidelines.md), it's correct that you need two owners, but contrib owners are the one who is responsible for the content in this directory that can make changes on behalf of BentoML.

Would it be more appropriate it add yourself here?

Reply:
Happy to add myself to the list.